prime-select.sh

```diff
@@ -219,12 +232,12 @@ function apply_current {
             # this can happen if user set nvidia but changed to "Integrated only" in BIOS (possible on some MUXED Optimus laptops)
             # in that case the NVIDIA card is not visible to the system and we must switch to intel
             
-            logging "Forcing intel due to NVIDIA card not found"
-            current_type="intel"
+            logging "Forcing intel due to NVIDIA card not found" current_type="intel"
```

Comment:
logging call and setting current_type in the same line?

Reply:
This hunk shouldn't be there